app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt

```diff
@@ -1104,15 +1104,19 @@ class ViewPagerActivity : SimpleActivity(), ViewPager.OnPageChangeListener, View
     }
 
     private fun askConfirmDelete() {
-        val filename = "\"${getCurrentPath().getFilenameFromPath()}\""
+        val path = getCurrentPath()
+        val fileDirItem = FileDirItem(path, path.getFilenameFromPath(), getIsPathDirectory(path))
```

Comment:
please use the helper function File.toFileDirItem() for creating FileDirItem from path

Reply:
Changes made as requested